<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRectDialog</class>
 <widget class="QDialog" name="EditRectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Rectangle</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>155</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="xLabel">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="xLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="yLabel">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="yLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="widthLabel">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="widthLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="heightLabel">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="heightLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="velocityLabel">
      <property name="text">
       <string>Velocity </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="velocityLineEdit"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>xLineEdit</tabstop>
  <tabstop>yLineEdit</tabstop>
  <tabstop>widthLineEdit</tabstop>
  <tabstop>heightLineEdit</tabstop>
  <tabstop>velocityLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
